test_vm/vendor/cookbooks/dbfit_test/recipes/sqlserver.rb

```diff
@@ -0,0 +1,44 @@
+remote_file '/etc/yum.repos.d/mssql-server.repo' do
+  source 'https://packages.microsoft.com/config/rhel/7/mssql-server.repo'
+  action :create
+end
+
+package 'mssql-server'
+
+ENV['SA_PASSWORD'] = 'DbFit000!'
```

Comment:
We're using the same password in multiple places, can we reuse it at least within the same script?